test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -0,0 +1,169 @@
+defmodule Membrane.HTTPAdaptiveStream.SinkBinIntegrationTest do
+  use ExUnit.Case, async: true
+
+  import Membrane.Testing.Assertions
+  alias Membrane.Testing
+
+  @doc """
```

Comment:
Why does it have `@doc` if it is not meant for docuemntation generation? Just to use multiline string?

Reply:
You're right. I am still learning commercial elixir programming standards and I didn't know using @doc is not appropriate for longer comments. I will introduce this change.